teleport-node/Dockerfile

```diff
@@ -0,0 +1,19 @@
+# Stage1: build from source
+FROM quay.io/cybozu/golang:1.15-focal AS build
+
+ARG TELEPORT_VERSION=6.0.0
```

Comment:
The latest is 6.0.1
https://github.com/gravitational/teleport/releases/tag/v6.0.1